src/winston2.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as is from '@sindresorhus/is';
+
+import * as common from './common';
```

Comment:
Suggestion: `import {LogginCommon, LOGGING_TRACE_KEY} from './common';`

Reply:
because i have to export LOGGING_TRACE_KEY i'll have to `LOGGING_TRACE_KEY as _LOGGING_TRACE_KEY` which is less nice thahn pulling from common.LOGGING_TRACE_KEY imo but ill go for it